lib/tabula.js

```diff
@@ -78,6 +88,12 @@ function sortArrayOfObjects(items, fields, options) {
                 a_cmp = a_field;
                 b_cmp = b_field;
             }
+
+            // Use user-supplied comparator function if supplied
+            if (typeof (obj.comparator) === 'function') {
```

Comment:
Just use `if (obj.comparator)` here. The assert.optionalFunc above already ensures it is a function.